components/persistor/lib/types/PersistorTransaction.ts

```diff
@@ -1,9 +1,18 @@
 export type PersistorTransaction = {
-    id: number,
-    dirtyObjects: object,
-    savedObjects: object,
-    touchObjects: object,
-    deletedObjects: object,
-    remoteObjects: Map<string, string> // identifiers for objects not stored directly in our db. key -> S3VersionId
-    deleteQueries: {}
+    updateConflict?: boolean;
+    id: number;
+    dirtyObjects: object;
+    savedObjects: object;
+    touchObjects: object;
+    deletedObjects: object;
+    remoteObjects: Map<string, string>; // identifiers for objects not stored directly in our db
```

Comment:
should this be a tsdoc comment?

Reply:
added as a legit comment with some more detail on what's actually stored here